curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md

```diff
@@ -21,22 +21,23 @@ In the text of your `p` element, turn the words `cat photos` into a link by addi
 You should nest a new anchor (`a`) element within the `p` element.
 
 ```js
-assert($('p > a').length);
+const nestedAnchor = document.querySelector(`p > a`);
+assert(nestedAnchor);
```

Comment:
Using `assert.isNotNull()` might be more correct here.

Reply:
right ... I have updated this